src/containers/AccountConnection.jsx

```diff
@@ -214,33 +232,10 @@ class AccountConnection extends Component {
               onCancel={() => this.cancel()}
             />
           </div>
-          <div className={styles['col-account-connection-data']}>
```

Comment:
It is used in src/containers/AccountConnection.jsx and src/containers/AccountSuccessTimeout.jsx.